src/main/java/frc/robot/commands/BalancerCommand.java

```diff
@@ -6,24 +6,29 @@
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 import frc.robot.subsystems.drivetrain.BaseDrivetrain;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
 import edu.wpi.first.wpilibj.Timer;
 
 public class BalancerCommand extends CommandBase {
-    private final BaseDrivetrain driveSubsystem;
+    private final BaseSwerveSubsystem driveSubsystem;
```

Comment:
The point of abstracting a "go forward with power" method between tank and swerve was so that we could run this command with both, right? I don't think restricting it to a swerve subsystem is necessary.